db/models/Entry.js

```diff
@@ -5,6 +5,8 @@ const { Schema } = mongoose;
 const entrySchema = new Schema(
   {
     userId: { type: [Schema.Types.ObjectId], ref: "User" },
+    category: { type: [Schema.Types.ObjectId], ref: "Category" },
+    categoryName: { type: String },
```

Comment:
I wonder if you want to store the categoryName here as well, since you now have multiple attributes that refer to the category. Normally you would get the category name from the category document after the entry got populated by mongoDB. Maybe you don't need this attribute at all.

Reply:
I didn't have this before, but then I would have to call the api for the categories for rendering the word list items (because they show the category and the category id is not very visual :D - So I added it to the entry object so I can render it)